src/database/standalone-sqlite.ts

```
@@ -3416,6 +3438,10 @@ if (!isMainThread) {
           worker.saveDataContentAttributes(args[0]);
           parentPort?.postMessage(null);
           break;
+        case 'getBundle':
+          const bundle = worker.getBundle(args[0]);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)